projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-row.component.ts

```diff
@@ -137,6 +137,15 @@ export class IgxPivotRowComponent extends IgxRowDirective implements OnChanges {
         return this.grid.pivotConfiguration.values.find(v => v.member === measureName)?.styles;
     }
 
+    public isCellActive(visibleColumnIndex) {
```

Comment:
There might be an issue here since it seems that 2 cells get active:
![navigation-piv](https://user-images.githubusercontent.com/10397980/144054751-02b72f78-da29-4937-b95b-4386559d9079.gif)